src/test/scala/chiselTests/Vec.scala

```diff
@@ -3,79 +3,14 @@
 package chiselTests
 
 import circt.stage.ChiselStage
+import circt.stage.ChiselStage.emitCHIRRTL
 import chisel3._
 import chisel3.testers.{BasicTester, TesterDriver}
 import chisel3.util._
 import org.scalacheck._
```

Comment:
Do these changes remove the dependency on scalacheck in this test or globally?

Reply:
Just in this test, but it's still a step toward getting rid of it!